lighthouse-core/lib/log.js

```diff
@@ -32,13 +32,7 @@ const colors = {
 };
 
 // whitelist non-red/yellow colors for debug()
-if (isBrowser) {
-  const debugBrowser = require('debug/browser');
-  debugBrowser.colors = [colors.cyan, colors.green, colors.blue, colors.magenta];
-} else {
-  const debugNode = require('debug/node');
-  debugNode.colors = [colors.cyan, colors.green, colors.blue, colors.magenta];
-}
+debug.colors = [colors.cyan, colors.green, colors.blue, colors.magenta];
```

Comment:
another option is to lock the `package.json` version to `debug` 2.2.0 as our `yarn.lock` file does